config/provider.go

```diff
@@ -182,38 +150,19 @@ func (p WithEC2IMDSRegion) GetRegion() (string, error) {
 	return result.Region, nil
 }
```

Comment:
Intentionally remove the `WithEC2IMDSRegion` functional option helper? Maybe this should be `WithUseEC2IMDSRegion`? to imply its behavior?

Reply:
The `With..` helpers were moved to config.go. This was to help improve visibility in code, and bundle them with `LoadOptions` which are defined in config.go .